kubernetes/ansible/k8s-install.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: "Install kubernetes"
+  hosts: "{{ override_host | default('masters') }}"
+  gather_facts: "{{ gathering_host_info | default('true') | bool == true }}"
+
+  # Define default values for variables that are common between roles
+  vars:
+    client_tool: kubectl
+
+  tasks:
+    - name: "Execute containerd installation role"
+      import_role:
+        name: 'containerd'
+      vars:
+        state: present
+      tags: [containerd]
+    - name: "Execute docker installation role"
+      import_role:
+        name: 'docker'
+      vars:
+        state: present
+      tags: [docker]
+    # - name: "Execute docker configuration role"
+    #   import_role:
+    #     name: 'k8s_docker'
+    #   tags: [docker]
```

Comment:
To be removed if not used

Reply:
removed as the docker configuration role, when applied, prevents docker from starting